src/redux/actions/coursesActions.js

```diff
@@ -0,0 +1,93 @@
+import axiosWithAuth from '../../utils/axiosWithAuth';
+
+export const EDITING = 'EDITING';
+export const ADD_COURSE = 'ADD_COURSE';
+export const GET_COURSES = 'GET_COURSES';
+export const UPDATE_COURSE = 'UPDATE_COURSE';
+export const DELETE_COURSE = 'DELETE_COURSE';
+export const CANCEL_EDIT = 'CANCEL_EDIT';
+
+export const setEditing = id => {
+  return { type: EDITING, payload: id };
+};
+
+export const cancelEdit = () => {
+  return { type: CANCEL_EDIT };
+};
+
+export const getCourses = idToken => async dispatch => {
+  try {
+    const res = await axiosWithAuth(idToken).get(`/courses`);
+    dispatch({
+      type: GET_COURSES,
+      payload: res.data,
+    });
+  } catch (error) {
+    dispatch({
+      payload: error.message,
+    });
```

Comment:
It looks like this code is dispatching an action without a type, so it isn't doing anything. Do we need to add an action type for errors and a case in the switch inside the reducer?